### PR TITLE
Introduce html-based AboutGuide.  Normalize dialogs. Use theme.

### PR DESCRIPTION
**Pull request by pablo-mayrgundter**
Use some Mui dialog components, more typography, theme support, revive dialog header icons.  Move About into subfolder.

### CODE REVIEW

**Comment by OlegMoshkovich** [comment]
I don't know what state of work this is, but on the first glance a lot of elements need to be edited.

**Comment by OlegMoshkovich** [comment]
image
the dialogs don't look good with the headers.

**Comment by OlegMoshkovich** [comment]
why is this underline?
image

**Comment by OlegMoshkovich** [comment]
image
this is a wrong icon.

**Comment by OlegMoshkovich** [comment]
image
b is enormous and the logo is image in the middle is broken.

**Comment by OlegMoshkovich** [comment]
image
share and about icons are smaller.

**Comment by pablo-mayrgundter** [comment]
> image
> 
> this is a wrong icon.

Yep, fixed.

**Comment by OlegMoshkovich** [comment]
Take a look at the design dialogs
[here](https://www.figma.com/file/CFu1IwNXkyh7pP5Helh3ye/Design-Sprints?node-id=260%3A4473&t=aC8AlsiUAt5wEqTU-0)

**Comment by pablo-mayrgundter** [comment]
> image
> 
> the dialogs don't look good with the headers.

Agreed.. I like the icon though.  See what you think about moving them to left instead of above.

**Comment by pablo-mayrgundter** [comment]
> image
> 
> share and about icons are smaller.

Fixed.

**Comment by OlegMoshkovich** [comment]
@pablo-mayrgundter can the operations group go back to this format?
image

**Comment by OlegMoshkovich** [comment]
image
 B disappeared from the left corner.

**Comment by OlegMoshkovich** [comment]
image

this is still not balanced. B is too big.
mixed feelings about bldrs.ai underneath.

image
if you want to keep BLDRS icon in the guide graphic, it should be centered.

**Comment by OlegMoshkovich** [comment]
image
there is too much space between the button and the dialog content.
the vertical scroll bar should not be displayed.

**Comment by OlegMoshkovich** [comment]
image
all of the scroll bars are green :) 
did you connect it to the theme?

**Comment by OlegMoshkovich** [comment]
image

**Comment by OlegMoshkovich** [comment]
@pablo-mayrgundter dialog close buttons do not work on mobile.

**Comment by pablo-mayrgundter** [comment]
PTAL

**Comment by OlegMoshkovich** [comment]
Notes Icon does not work.

**Comment by OlegMoshkovich** [comment]
Did slight adjustments to the graphics on on the about dialog.
image

**Comment by pablo-mayrgundter** [comment]
> Did slight adjustments to the graphics on on the about dialog. image

done.

**Comment by pablo-mayrgundter** [comment]
> Notes Icon does not work.

done

**Comment by pablo-mayrgundter** [comment]
PTAL

**Review by OlegMoshkovich — APPROVED** [review]

File: src/Theme.jsx [hunk 1]
```diff
@@ -171,13 +176,39 @@ function loadTheme(mode) {
         disableRipple: true,
       },
     },
+    MuiToggleButton: {
+      styleOverrides: {
+        sizeMedium: {
+          'margin': '1em',
```
[the remark this answers is not in the document]
Comment (pablo-mayrgundter) [reply src/Theme.jsx]: Done

File: src/Components/Dialog.jsx [hunk 2]
```diff
@@ -1,52 +1,86 @@
...
+              marginRight: '0.5em',
+            },
+          }}
+        >
+          {icon && icon} {headerText}
         </Typography>
+        {content}
       </DialogContent>
-    </MuiDialog>)
+      <DialogActions sx={{overflowY: 'hidden'}}>
```
[the remark this answers is not in the document]
Comment (pablo-mayrgundter) [reply src/Components/Dialog.jsx]: Done

**Pull request merged**